assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -115,6 +148,25 @@ function ReaderRevenueManagerSetupCTABanner( { Widget, WidgetNull } ) {
 		}
 	}, [ showBanner, viewContext ] );
 
+	if ( isTooltipVisible ) {
+		return (
+			<Fragment>
+				<WidgetNull />
+				<AdminMenuTooltip
+					title=""
+					content={ __(
+						'You can always enable reader revenue manager from Settings later',
```

Comment:
Whoops - I completely missed this, "Reader Revenue Manager" should have capital initials.

I'll apply the change myself to save the round trip.

```suggestion
						'You can always enable Reader Revenue Manager from Settings later',
```